tests/test_errors.py

```diff
@@ -93,7 +93,7 @@ def my_function(input_value, info):
         v.validate_python(42)
 
     assert str(exc_info.value) == (
-        '1 validation error for function-plain[my_function()]\n'
+        '1 validation error for validator-function-plain[my_function()]\n'
```

Comment:
I guess we should just double-check we're happy with this new format, as it's potentially breaking to downstream test suites and if we change this we should aim to do so once and not again?

(e.g. would it be better if this just read `1 validation error for my_function()`?)